### PR TITLE
Fixing problems in our rst docs

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #7090

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: docs/contributing/architecture/rest-api.rst [hunk 1]
```diff
@@ -157,15 +157,15 @@ control over what fields are exposed in the REST API, and that those fields are
...
 
-	>>> from serializers import RepositorySerializer
-	>>> RepositorySerializer()
-	RepositorySerializer():
-		pulp_href = HyperlinkedIdentityField(view_name='repositories-detail')
-		name = CharField(style={'base_template': 'textarea.html'}, validators=[<UniqueValidator(queryset=Repository.objects.all())>])
-		description = CharField(allow_blank=True, required=False, style={'base_template': 'textarea.html'})
-		last_content_added = DateTimeField(allow_null=True, required=False)
-		last_content_removed = DateTimeField(allow_null=True, required=False)
-		content = HyperlinkedRelatedField(many=True, read_only=True, view_name='content-detail')
```
Comment (daviddavis) [inline docs/contributing/architecture/rest-api.rst:168]: Fixed use of tabs here

File: CHANGES.rst [hunk 2]
```diff
@@ -750,6 +751,3 @@ Features
 
 - Added a new required field called CONTENT_TYPES to the Repository model.
   `#5701 <https://pulp.plan.io/issues/5701>`_
-
-
-----
```
Comment (daviddavis) [inline CHANGES.rst:755]: `Document may not end with a transition.`

File: docs/contributing/architecture/rest-api.rst [hunk 3]
```diff
@@ -383,7 +383,7 @@ custom filter based on the `BaseInFilter`.
             pass
 
         class RepositoryFilter(filters.FilterSet):
-        name_in_list = CharInFilter(name='name', lookup_expr='in')
+            name_in_list = CharInFilter(name='name', lookup_expr='in')
```
Comment (daviddavis) [inline docs/contributing/architecture/rest-api.rst:386]: Bad python.

File: docs/plugins/api-reference/storage.rst [hunk 4]
```diff
@@ -3,4 +3,4 @@
 pulpcore.plugin.storage
 =======================
 
-.. automodule:: pulpcore.plugin.storage
\ No newline at end of file
+.. automodule:: pulpcore.plugin.storage
```
Comment (daviddavis) [inline docs/plugins/api-reference/storage.rst:6]: No newline.

File: docs/plugins/plugin-writer/concepts/subclassing/models.rst [hunk 5]
```diff
@@ -55,7 +55,7 @@ can make and run database migrations with:
 
 .. code-block:: bash
 
-      pulpcore-manager makemigrations <plugin_app_label>
+      pulpcore-manager makemigrations plugin_app_label
```
Comment (daviddavis) [inline docs/plugins/plugin-writer/concepts/subclassing/models.rst:58]: Invalid bash.

Comment (daviddavis) [reply docs/plugins/plugin-writer/concepts/subclassing/models.rst:58]: I could also make this code block not be bash.

File: docs/plugins/reference/how-to-doc-api.rst [hunk 6]
```diff
@@ -32,8 +32,8 @@ You may disable schema generation for a view by setting ``schema`` to ``None``:
 .. code-block:: python
 
     class CustomView(APIView):
-    ...
-    schema = None  # Will not appear in schema
+        # ...
+        schema = None  # Will not appear in schema
```
Comment (daviddavis) [inline docs/plugins/reference/how-to-doc-api.rst:36]: Invalid python.

File: docs/workflows/repairing-repository-versions.rst [hunk 7]
```diff
@@ -22,7 +22,7 @@ got corrupted in some way (i.e. bit rot).
 To start a repair task on a specific repository version, send a POST request to its `repair`
 endpoint:
 
-.. code-block:: bash
+.. code-block::
```
Comment (daviddavis) [inline docs/workflows/repairing-repository-versions.rst:25]: This code block is not valid bash.

File: docs/plugins/plugin-writer/concepts/subclassing/models.rst (hunk 5 above)
Comment (mdellweg) [reply docs/plugins/plugin-writer/concepts/subclassing/models.rst:58]: What about
```suggestion
      pulpcore-manager makemigrations $PLUGIN_APP_LABEL
```

**Comment by fao89** [comment]
Nice!

**Pull request merged**